### PR TITLE
feat: Added GTM tag to 3DVV

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Adds the Google Tag Manager snippet to the 3DVV for analytics and the cookie banner. Closes https://github.com/allen-cell-animated/volume-viewer/issues/229.

*Estimated review size: small, 5 minutes*

### CODE REVIEW

**Review by interim17 — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -1,29 +1,61 @@
...
+      (function (w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != "dataLayer" ? "&l=" + l : "";
+        j.async = true;
+        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, "script", "dataLayer", "GTM-5J7562WW");
```
Comment (frasercl) [inline public/index.html:15]: That last argument must be some kind of API key? I'd ask if we needed to do something extra to keep it secret, but I guess this is likely auto-generated anyways and will appear in the page source regardless.

File: public/index.html [hunk 2]
```diff
@@ -1,29 +1,61 @@
...
+  <body style="margin: 0">
+    <!-- Google Tag Manager (noscript) -->
+    <noscript
+      ><iframe
+        src="https://www.googletagmanager.com/ns.html?id=GTM-5J7562WW"
+        height="0"
+        width="0"
+        style="display: none; visibility: hidden"
+      ></iframe
+    ></noscript>
```
Comment (frasercl) [inline public/index.html:56]: what the heck is going on with the formatting of these tags here?

**Review by frasercl — APPROVED** [review]
Looks good!

File: public/index.html (hunk 1 above)
Comment (ShrimpCryptid) [reply public/index.html:15]: Yup, it's an identifier for Google Tag Manager. Afaik there's no reason to keep it private.

File: public/index.html (hunk 2 above)
Comment (ShrimpCryptid) [reply public/index.html:56]: I'm honestly shocked that Prettier didn't decide to intervene... fixed!

**Pull request merged**